uui-core/src/hooks/useVirtualList/useVirtualList.ts

```diff
@@ -53,6 +57,20 @@ export function useVirtualList<List extends HTMLElement = any, ScrollContainer e
         rowsSelector,
     ]);
 
+    useLayoutEffectSafeForSsr(() => {
+        if (scrollContainer.current?.clientHeight
+            && prevScrollContainerClientHeight
+            && scrollContainer.current?.clientHeight !== prevScrollContainerClientHeight
+        ) {
+            ++scrollContainerHeightChangesCount.current;
+        }
+        if (scrollContainerHeightChangesCount.current > 20 && !scrollContainerHeightIsNotLimited.current) {
+            scrollContainerHeightIsNotLimited.current = true;
+            console.error(`The scroll container height is unlimited, likely because the parent container's height is set to 100%. Please ensure that the VirtualList's parent container has a defined, limited height.
+It is recommended to use viewport height (vh) units to set a relative height.`);
```

Comment:
Let's not guess for reasons here. It can be misleading. Just `The scroll container height is not limited. Please ensure that the VirtualList's parent container has a defined, limited height.` - will be good enough.

Also, it might be a good idea to provide a link to the VirtualList docs.